RELEASE.md

```diff
@@ -33,3 +33,50 @@ We don't use changesets prerelease mode yet as we want to avoid additonal steps
 ### **Next**
 
 - Use Release Snapshot to publish new version for testing on `@next` before releasing to production
+
+
+### Patching older major/minor versions
+Backporting a fix to an earlier version is rarely necessary and should generally be avoided. However, if required, the following steps outline how to proceed.
+
+#### Backporting Workflow
+Say we have released 1.0.1 and 1.1.0 on main, and we want to backport a patch to the 1.0 version.
```

Comment:
Maybe use 2.0.0 and 1.1.0 in this example? I don't think we'll need to backport patches between minor versions, its most common between major versions.

Reply:
Sure 👍